packages/cashier/src/pages/account-transfer/account-transfer-no-account/account-transfer-no-account.tsx

```diff
@@ -1,21 +1,19 @@
 import React from 'react';
 import { Icon, Button, Text } from '@deriv/components';
-import { getPlatformSettings } from '@deriv/shared';
+import { getPlatformSettings, routes } from '@deriv/shared';
 import { Localize } from '@deriv/translations';
 import { useStore, observer } from '@deriv/stores';
 import './account-transfer-no-account.scss';
+import { useHistory } from 'react-router-dom';
 
-type TAccountTransferNoAccount = {
-    openAccountSwitcherModal?: () => void;
-};
-
-const AccountTransferNoAccount = observer(({ openAccountSwitcherModal }: TAccountTransferNoAccount) => {
```

Comment:
@rostislav-deriv `openAccountSwitcherModal` prop has been removed, So we need to also remove it wherever this component has been used.